### PR TITLE
revert rebase changes on update

### PR DESCRIPTION
**Pull request by xabierlaiseca**
What
----
When rebasing to newer version some update changes were reverted.

References
----------
PR that was reverted: https://github.com/confluentinc/terraform-provider-rancher2/pull/7
Commit that reverted the changes: https://github.com/confluentinc/terraform-provider-rancher2/commit/983b41073dcf5805134e80ff1151e73cb2b02055

### CODE REVIEW

File: rancher2/resource_rancher2_cluster.go [hunk 1]
```diff
@@ -327,6 +338,12 @@ func clusterStateRefreshFunc(client *managementClient.Client, clusterID string)
 			return nil, "", err
 		}
 
+		// If we encounter an error, such as "only one of zone or region must be specified"
```
Comment (dstockton) [inline rancher2/resource_rancher2_cluster.go:341]: Is this our code? It's not in upstream

**Comment by isiachi** [comment]
Is this a partial revert of 983b410?

**Comment by xabierlaiseca** [comment]
@isiachi when rebasing some changes were reverted, I'm getting them back

File: rancher2/resource_rancher2_cluster.go (hunk 1 above)
Comment (xabierlaiseca) [reply rancher2/resource_rancher2_cluster.go:341]: yes, it is https://github.com/confluentinc/terraform-provider-rancher2/pull/6

Comment (dstockton) [reply rancher2/resource_rancher2_cluster.go:341]: Looks like I also binned this off after @thomas-maurice added it...
https://github.com/confluentinc/terraform-provider-rancher2/commit/aac66e479ccd958e937bf614056299367f9d7f0f

🤦‍♂

**Review by dstockton — APPROVED** [review]
I know a few of us have been over this but I've also been through it comparing with the original PRs and what's there now.
Thanks for catching these & raising the PR to sort it all out!
👍

**Pull request merged**